doc/nrf/migration/migration_guide_1.x_to_2.x.rst

```diff
@@ -0,0 +1,415 @@
+.. _ncs_2.0.0_migration:
+
+Migration notes for |NCS| v2.0.0
+################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+In version 2.0.0, the |NCS| introduced a number of breaking changes into its codebase.
+If your application was built using |NCS| v1.x.x, make sure to complete the actions listed in the following sections to migrate your application to |NCS| v2.x.x.
+
+Toolchain change
+****************
+
+|NCS| no longer uses the GNU Arm Embedded Toolchain.
+The current default toolchain is the Zephyr toolchain.
+
+*Required action*: Install the new toolchain by following the steps in "Install a Toolchain" in Zephyr's :ref:`zephyr:getting_started`.
```

Comment:
```suggestion
*Required action*: If you don't use the Toolchain Manager, install the Zephyr SDK by following the steps in "Install a Toolchain" in Zephyr's :ref:`zephyr:getting_started`. The updated Toolchain Manager already contains it.
```

Reply:
@martingalvan-nordic please double-check this^^

Reply:
It looks fine, though the Zephyr Getting Started guide involves running the SDK's setup.sh script. I'm not sure if that's correct in our case, since it ends up registering the SDK's toolchain within the CMake machinery and installs a set of "hosttools" which we don't use (e.g. Qemu, OpenOCD. etc).

For example, in the toolchain-manager we don't run that script; instead we simply set the `ZEPHYR_SDK_INSTALL_DIR` environment variable (as documented at https://docs.zephyrproject.org/2.7.0/getting_started/installation_linux.html#install-the-zephyr-software-development-kit-sdk).

Reply:
If this is the case, then we need to change our 'Install a toolchain' section in the manual installation instructions, because right now we are basically copying what is in Zephyr, after switching to Zephyr SDK.

Reply:
Both running the `setup.sh` or setting `ZEPHYR_SDK_INSTALL_DIR` have the same effect of the toolchain being found by the zephyr build system. Since this section is only relevant for those **not** using the Toolchain Manager then this is fine as it is.